occurrences/tests/test_models.py

```diff
@@ -42,3 +42,26 @@ def test_enrolment_creation(mock_get_event_data):
     EnrolmentFactory()
     assert Occurrence.objects.count() == 1
     assert StudyGroup.objects.count() == 1
+
+
+@pytest.mark.django_db
+def test_occurrence_seat_taken():
+    EnrolmentFactory()
+    occurrence = Occurrence.objects.all()[0]
```

Comment:
you can get the `enrolment = EnrolmentFactory()`, then `ocurrence = enrolment.occurrence` and `study_group=enrolment.study_group`. That's look cleaner IMO

Reply:
You are absolutely right!